Signal/src/ViewControllers/LongTextViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+import SignalServiceKit
+import SignalMessaging
+
+@objc
+public class LongTextViewController: OWSViewController {
```

Comment:
I added a dedicated view to show "long text".  Pretty simple.  Has a share button.

Reply:
Yup, closed that issue.